py-polars/tests/unit/namespaces/test_datetime.py

```diff
@@ -144,8 +144,18 @@ def test_local_datetime_sortedness(time_zone: str | None, expected: bool) -> Non
 def test_local_time_sortedness(time_zone: str | None) -> None:
     ser = (pl.Series([datetime(2022, 1, 1, 23)]).dt.replace_time_zone(time_zone)).sort()
     result = ser.dt.time()
-    assert result.flags["SORTED_ASC"] is False
-    assert result.flags["SORTED_DESC"] is False
+    assert result.flags["SORTED_ASC"]
+    assert not result.flags["SORTED_DESC"]
```

Comment:
this one should've been sorted to begin with, but that's already being dealt with in https://github.com/pola-rs/polars/pull/13808